memsimplekv/keyvalue.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package memsimplekv
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	errgo "gopkg.in/errgo.v1"
+
+	"github.com/juju/simplekv"
+)
+
+// NewStores returns a new Stores instance.
+// All Store instances returned from a Stores will share
```

Comment:
This part of the comment is no longer needed (or true, or meaningful)

Reply:
done